data/brands/amenity/bicycle_rental.json

```diff
@@ -1060,16 +1060,17 @@
       }
     },
     {
-      "displayName": "Citybike Wien",
-      "id": "citybikewien-cda769",
+      "displayName": "WienMobil Rad",
       "locationSet": {"include": ["at"]},
       "tags": {
         "amenity": "bicycle_rental",
-        "brand": "Citybike Wien",
-        "brand:wikidata": "Q14541491",
-        "network": "Citybike Wien",
-        "network:wikidata": "Q14541491",
-        "operator": "Citybike Wien"
+        "brand": "WienMobil Rad",
+        "brand:wikidata": "Q111794110",
+        "network": "WienMobil Rad",
+        "network:wikidata": "Q111794110",
+        "operator:wikidata": "Q2351279",
+        "operator": "Nextbike",
+        "operator:wikidata": "en:Nextbike"
```

Comment:
```suggestion
```

Hey! looks like you are trying to add a wikipedia? After discussion in #6481 NSI finally stop suggest wikipedia content so this is not needed. Thank you~